examples/laravel/tests/dredd/hooks/hookfile.php

```diff
@@ -1,15 +1,21 @@
 <?php
 
 use Dredd\Hooks;
-use Illuminate\Support\Facades\Artisan;
 
 require __DIR__ . '/../../../vendor/autoload.php';
 
 $app = require __DIR__ . '/../../../bootstrap/app.php';
 
 $app->make(\Illuminate\Contracts\Console\Kernel::class)->bootstrap();
 
+Hooks::beforeAll(function (&$transaction) {
+    Illuminate\Support\Facades\Artisan::call('migrate:refresh', ['--seed' => true]);
+});
+
 Hooks::beforeEach(function (&$transaction) use ($app) {
+    // Beware this transaction only works for the hooks itself.
+    // If the API tests generated by Dredd make changes to the database,
+    // they will not be rolled back. In that case, you must use migrations.
     $app->make('db')->beginTransaction();
```

Comment:
I think its great to explain that transactions can't be easily used, however, I think we should remove this usage entirely from the example.

Reply:
How about we put information about it in the Readme instead? I think it may be a very common mistake that people could make - and as i had to experience, it costs a lot of time to figure out.

Reply:
Sorry I'm finally back from my traveling.  I meant that I don't think the example should use transactions.  I agree the README is the best place to put this information but to warn people and then give them an example with txns is just misleading.